doc/source/ray-air/doc_code/checkpoint_usage.py

```diff
@@ -0,0 +1,130 @@
+# flake8: noqa
+# isort: skip_file
+
+# __checkpoint_quick_start__
+from ray.train.tensorflow import to_air_checkpoint
+import tensorflow as tf
+
+# This can be a trained model.
+def build_model() -> tf.keras.Model:
+    model = tf.keras.Sequential(
+        [
+            tf.keras.layers.InputLayer(input_shape=(1,)),
+            tf.keras.layers.Dense(1),
+        ]
+    )
+    return model
+
+
+model = build_model()
+
+checkpoint = to_air_checkpoint(model)
```

Comment:
Feels a bit clunky, why not `checkpoint = model.to_checkpoint()`?

Reply:
there is no "model" class in Air.